ios/scripts/asc_bootstrap_app_availability.py

```
@@ -0,0 +1,213 @@
+        url = path_or_url
+        parsed_base = urllib.parse.urlsplit(base)
+        parsed_url = urllib.parse.urlsplit(url)
+        if (parsed_url.scheme, parsed_url.netloc) != (parsed_base.scheme, parsed_base.netloc):
+            raise RuntimeError("App Store Connect pagination changed API origin")
+
+    data = None if payload is None else json.dumps(payload).encode("utf-8")
+    request = urllib.request.Request(url, data=data, method=method)
+    request.add_header("Authorization", "Bearer " + token)
+    request.add_header("Content-Type", "application/json")
+    try:
+        with urllib.request.urlopen(request, timeout=30) as response:
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)